packages/react/src/primitives/Menu/Menu.tsx

```diff
@@ -1,17 +1,19 @@
 import * as React from 'react';
 import classNames from 'classnames';
-import {
-  DropdownMenu,
-  DropdownMenuTrigger,
-  DropdownMenuContent,
-} from '@radix-ui/react-dropdown-menu';
+import * as Dropdown from '@radix-ui/react-dropdown-menu';
 
 import { ButtonGroup } from '../ButtonGroup';
 import { ComponentClassNames } from '../shared/constants';
 import { IconMenu } from '../Icon/internal';
 import { MenuButton } from './MenuButton';
 import { MenuProps, Primitive } from '../types';
 
+let sanitizedDropdown = { default: undefined, ...Dropdown };
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+sanitizedDropdown = sanitizedDropdown.default ?? sanitizedDropdown;
+const { DropdownMenu, DropdownMenuTrigger, DropdownMenuContent } =
+  sanitizedDropdown;
```

Comment:
Just curious, is this eslint comment avoidable?

Reply:
It does not allow me to do that because there is a change `sanitizedDropdown.default` is `any`

Reply:
To add to this point, writing in this way is the less eslint comment way I found 😈

Reply:
Understood. Just one more question for my understanding, when will `sanitizedDropdown.default` be defined vs undefined? 

That conditional was confusing to me, because I was thinking that it would always be one or the other.

Reply:
When that radix module was imported as CJS(e.g. SSR), the default will be defined. Its meaningful content will be place into `default` when doing a namespace import compared to the ESM.

Reply:
Got it, makes sense to me now, thanks

Reply:
@zchenwei Can we add a comment for this? I think your explanation is great, but would like to help out future us who are confused. :) We could also add a comment linking the the radix issue which could allow us to removed this workaround?

Reply:
I think I already have a comment there. Refresh?